includes/class-linkbacks-mf2-handler.php

```diff
@@ -192,7 +192,12 @@ public static function generate_commentdata( $commentdata ) {
 				}
 
 				if ( isset( $author['url'] ) ) {
-					$commentdata['comment_meta']['semantic_linkbacks_author_url'] = $author['url'];
+					if ( in_array( $commentdata['comment_type'], array( 'webmention', '' ) ) ) {
```

Comment:
if we replace the author_url we break the comment detection/duplicate comment code of the web mention plugin: https://github.com/pfefferle/wordpress-webmention/blob/master/includes/class-webmention-receiver.php#L457